index.js

```diff
@@ -1,16 +1,16 @@
 const guessInput = document.getElementById('guess');
 const submitButton = document.getElementById('submit');
 const resetButton = document.getElementById('reset');
-const messages = document.getElementsByClassName('message');
+const messages = document.querySelectorAll('.message');
 const tooHighMessage = document.getElementById('too-high');
 const tooLowMessage = document.getElementById('too-low');
 const maxGuessesMessage = document.getElementById('max-guesses');
-const numberOfGuessesMessage = document.getElementById('num-of-guesses');
+const numberOfGuessesMessage = document.getElementById('number-of-guesses');
 const correctMessage = document.getElementById('correct');
 
 let targetNumber;
-const attempts = 0;
-const maxNumberOfAttempts = 5;
+let attempts = 0;
+let maxNumberOfAttempts = 5;
```

Comment:
Nitpick: This `maxNumberOfAttempts` variable won't increase before, during, or after a game. The number of guesses allowed with always be set to 5. Therefore, this variable can set to a `const maxNumberOfAttempts = 5;`. 

For example, when a new game is started:
- guesses allowed (`maxNumberOfAttempts`): 5
  - this number stays the same, because you're always allowed 5 guesses per game
- guesses made (`attempts`): 0 -> 1 -> 2 -> 3 -> 4 -> 5 -> no more guesses are allowed
  - this number increases each time a guess is submitted (game is stopped when you've reached the limit of guesses allowed)
- guesses submitted (`guess`): 22 -> 45 -> ...etc
  - this number is changed each time a guess is submitted
- target number: 88 
  - this number is changed for each new game